rust/macros/module.rs

```diff
@@ -724,6 +732,149 @@ pub fn module_misc_device(ts: TokenStream) -> TokenStream {
     .expect("Error parsing formatted string into token stream.")
 }
 
+pub fn rtnl_link_ops(ts: TokenStream) -> TokenStream {
```

Comment:
Shouldn't this live in a separate module rather than `module.rs`?

Reply:
Could be. Mainly have it there because all helpers are defined there

Reply:
Extracted it into `mod net`. Also created `mod helpers` for all helper function (that I could identify as generic helpers).